pages/report-repair/[route].js

```diff
@@ -272,6 +273,22 @@ function ReportRepair() {
           ]}
         />
       )
+    case 'repair-shower-problems':
+      return (
+        <RepairProblemBestDescription
+          handleChange={handleChange}
+          values={values}
+          options = {[
+            { value: 'electricShowerUnit', title: 'Electric shower unit'},
+            { value: 'tapShower', title: 'Tap shower'},
```

Comment:
This value doesn't _match_ the value in [flow](https://github.com/City-of-Lincoln-Council/housing-repairs-online-frontend/pull/61/files#diff-4f43f537bc3e655255f7b417ad7e89ad1c24015fa0b2139cc7411ec1a3bb9419R56). Here it's `tapShower` and `showerTap` in flow.
Please change one of them to match - if they don't match the user isn't progressed onto the description page.

Reply:
well spotted!